python/tk_hiero_export/shot_updater.py

```diff
@@ -129,16 +129,24 @@ def taskStep(self):
         # get cut info
         handles = self._cutHandles if self._cutHandles is not None else 0
```

Comment:
Don't worry about the above. It's the only comment and isn't worth the effort since what's there works fine.